client/image.go

```diff
@@ -35,3 +38,28 @@ func (client *APIClient) ImageList() ([]types.Image, error) {
 	return imageList, err
 
 }
+
+// ImageRemoves deletes images.
+func (client *APIClient) ImageRemoves(name string, force bool) error {
+	q := url.Values{}
+	if force {
+		q.Set("force", "true")
+	}
+
+	resp, err := client.delete("/images/"+name, q)
+	if err != nil {
+		return err
+	}
+
+	defer ensureCloseReader(resp)
+
+	if resp.StatusCode != http.StatusNoContent {
```

Comment:
I think you defined a 2xx code to be 200 in swagger, but here you are comparing this with 204. Seems a little bit improper.